package.json

```
@@ -1,12 +1,12 @@
 {
-	"name": "@woocommerce/OpenAI-Product-Feed",
+	"name": "@woocommerce/Product-Feed-for-OpenAI",
 	"private": true,
-	"title": "OpenAI-Product-Feed",
+	"title": "Product-Feed-for-OpenAI",
```

We do not need dashes for title, right? 

```suggestion
	"title": "WooCommerce Product Feed",
```

---

We don't, removed in https://github.com/woocommerce/OpenAI-Product-Feed/pull/32/commits/08c962d50f9197bc38b2466d77c5e68266cb41ee